Src/BootCamp.Chapter/Program.cs

```diff
@@ -2,11 +2,11 @@
 
 namespace BootCamp.Chapter
 {
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
+	class Program
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4859834)